dotcom-rendering/src/web/components/ReaderRevenueLinks.test.tsx

```diff
@@ -10,6 +10,11 @@ const shouldHideSupportMessaging: {
 jest.mock('@root/src/web/lib/contributions', () => ({
 	shouldHideSupportMessaging: jest.fn(() => true),
 }));
+jest.mock('@guardian/libs', () => ({
```

Comment:
countryCode is a mandatory field for the header, so I've had to mock it in this test